app/src/controllers/projectDetails/sagas.js

```diff
@@ -0,0 +1,18 @@
+import { fetchDataAction } from 'controllers/fetch';
+import { put, select, all, takeEvery } from 'redux-saga/effects';
+import { projectIdSelector } from 'controllers/pages';
+import { URLS } from 'common/urls';
+import { FETCH_PROJECT_INFO, PROJECT_INFO_NAMESPACE } from './constants';
+
+function* fetchProjectInfo() {
+  const projectId = yield select(projectIdSelector);
+  yield put(fetchDataAction(PROJECT_INFO_NAMESPACE)(URLS.project(projectId)));
```

Comment:
Looks like the same info as in `controllers/project`. Please remove and use existing one.

Reply:
On the administration page, we need to get information for the selected project, but the current user's project should be saved.